keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -207,6 +245,49 @@ Since falling back to etcd won't work, we should fail the requests and rely on
 rate limiting to prevent cascading failure.  I.e. `Retry-After` HTTP header (for
 well behaved clients) and [Priority and Fairness](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md).
 
+For such situations we will provide users with following tools:
+* a dedicated `apiserver_watch_cache_read_wait` metric to detect a problem with
+  watch cache.
+* a per-request override via `X-DEBUG: NO-WATCH-CACHE` HTTP header for
+  debugging.
+* an escape hatch `--apiserver-consistent-list-from-cache` flag to disable the
+  feature.
+
+Metric `apiserver_watch_cache_read_wait` will measure wait time experienced by 
+reads for watch cache to become fresh. If user notices a latency request in
+they can use this metric to confirm that the issue is caused by watch cache.
+
+Per request override via an HTTP request header `X-DEBUG: NO-WATCH-CACHE`, 
```

Comment:
Right, but on the other hand we don't want to provide stability guarantees for debug level feature. I think we can skip designing the API during the KEP review as the final API shape will be mostly shaped by API review.

I will remove the details about how override will be implemented with note that it will be finalized during API review.